src/Method/Definition.php

```diff
@@ -206,9 +167,9 @@ public function getMethodHelp()
      *
      * @param  object $object
```

Comment:
as typehint has been added on the parameter this is redundant now

Reply:
Removed redundant @param and @return tags where applicable.